src/main/resources/application.yml

```diff
@@ -1,17 +1,12 @@
 spring:
   application:
     name: bookstore
-  datasource:
-    driver-class-name: com.mysql.cj.jdbc.Driver
-    url: jdbc:mysql://localhost:3306/bookstore?serverTimezone=UTC
-    username: book_user
-    password: book1234
```

Comment:
Don’t remove properties from the configuration file. Let’s  use variables from .env